Assets/StreamingAssets/WorldGen/DefaultStartArea.sav

```diff
@@ -2720,6 +2720,5 @@
       "Type": "power_cable"
     }
   ],
-  "RoomBehaviors": []
-  }
+  "RoomBehaviors": [],
```

Comment:
I assume this is an accidental addition (since the trailing comma would cause a JSON parsing error right?)